agent_issuance/src/offer/aggregate.rs

```diff
@@ -343,6 +421,7 @@ pub mod tests {
                 credential_configuration_ids: vec![],
                 grant_types: grant_types.clone(),
                 tx_code_constraints: None,
+                delivery_options: None,
```

Comment:
I think it would be good to have at least one test to include a `Some()` value. You already added a `delivery_options` fixture below but it is unused atm.